docs/state_encryption.md

```diff
@@ -36,6 +36,8 @@ In this section we describe the features that are out of scope for state and pla
 
 The primary goal of this feature is to protect state and plan files **at rest**. It is not the goal of this feature to protect other channels secrets may be accessed through, such as the JSON output. As such, it is not a goal of this feature to encrypt any output on the standard output, or file output that is not a state or plan file.
 
+Furthermore, it is not a goal of this feature to *authenticate* the plan file. It does not protect against, among others, replay attacks where a malicious actor replaces a current plan or state file with an old one.
```

Comment:
Wait, the encryption **does** do authentication (that is, that a message hasn't been tampered with). It doesn't protect against replay attacks though.

Reply:
It authenticates that the encryption is authenticated, but it does not authenticate that the plan file is the plan file you want to run. This may be in need of better wording.